tasks/task03/word_stats.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+texts = [
+   "Hello, World!",
+   "The world is mine",
+   "Hello, how are you?"
+]
+
+words =[]
+
+for i, sentence in enumerate(texts):
+    index = i
+    for w in re.sub("[^\w]", " ", str.lower(sentence)).split():
```

Comment:
by default \w is treated as a special character, escape sequence. So regex does not work correctly. Either escape backslash itself (e.g `"\\w"`) or use raw strings: `r"[^\w]"`